bcos-sync/bcos-sync/protocol/PB/BlockSyncMsgImpl.h

```diff
@@ -30,7 +31,10 @@ class BlockSyncMsgImpl : virtual public BlockSyncMsgInterface
 {
 public:
     using Ptr = std::shared_ptr<BlockSyncMsgImpl>;
-    BlockSyncMsgImpl() : m_syncMessage(std::make_shared<BlockSyncMessage>()) {}
+    BlockSyncMsgImpl() : m_syncMessage(std::make_shared<BlockSyncMessage>()) 
+    {
```

Comment:
Remove this, and set the version to `BlockSyncStatusMsg` with `createBlockSyncStatusMsg`.